assets/js/components/notifications/ZeroDataStateNotifications/GatheringDataNotification.js

```diff
@@ -45,31 +48,32 @@ export default function GatheringDataNotification( { title } ) {
 		trackEvent( eventCategory, 'dismiss_notification' );
 	}, [ eventCategory ] );
 
-	const gatheringDataWaitTime = '72';
-
 	return (
-		<BannerNotification
-			id="gathering-data-notification"
-			title={ title }
-			description={ sprintf(
-				/* translators: %s: the number of hours the site can be in a gathering data state */
-				__(
-					'It can take up to %s hours before stats show up for your site. While you’re waiting, connect more services to get more stats.',
-					'google-site-kit'
-				),
-				gatheringDataWaitTime
-			) }
-			format="small"
-			onView={ handleOnView }
-			dismiss={ __( 'OK, Got it!', 'google-site-kit' ) }
-			dismissExpires={ getTimeInSeconds( 'day' ) }
-			SmallImageSVG={ GatheringDataIcon }
-			onDismiss={ handleOnDismiss }
-			isDismissible
-		/>
+		gatheringDataWaitTime && (
```

Comment:
Instead of conditionally rendering `BannerNotification` with the `&&` operator, can we return `null` early if `gatheringDataWaitTime` isn't available like so:

```js
if ( ! gatheringDataWaitTime ) {
	return null;
}
```

Reply:
> Instead of conditionally rendering `BannerNotification` with the `&&` operator, can we return `null` early if `gatheringDataWaitTime` isn't available like so:
> 
> ```js
> if ( ! gatheringDataWaitTime ) {
> 	return null;
> }
> ```

@hussain-t I think that is a fair observation, I will make said change.